scrape.py

```diff
@@ -256,7 +255,8 @@ def _parse_columns(grouped, date_leave, date_return):
 				"emisions": f"Carbon emissions estimate: {co2_emission} kilograms. {emission}% emissions " if co2_emission and emission else "",
 				"price": f"NGN{price}"
 			})
-
+		df = pd.DataFrame(flight_data)
+		df.to_json("h.json", orient='records')
```

Comment:
currently there are more flights on the website than what it outputs, that's why i requested for a new scraper class that scrapes all the flights.